autotest/src/main/kotlin/no/nav/familie/tilbake/e2e/domene/builder/BehandleForeslåVedtakBuilder.kt

```diff
@@ -0,0 +1,44 @@
+package no.nav.familie.tilbake.e2e.domene.builder
+
+import no.nav.familie.tilbake.e2e.domene.dto.AvsnittDto
+import no.nav.familie.tilbake.e2e.domene.dto.Avsnittstype
+import no.nav.familie.tilbake.e2e.domene.dto.ForeslåVedtakDto
+import no.nav.familie.tilbake.e2e.domene.dto.FritekstavsnittDto
+import no.nav.familie.tilbake.e2e.domene.dto.PeriodeMedTekstDto
+import no.nav.familie.tilbake.e2e.domene.dto.UnderavsnittDto
+import no.nav.familie.tilbake.e2e.domene.dto.Underavsnittstype
+import no.nav.familie.tilbake.e2e.domene.dto.felles.PeriodeDto
+
+class BehandleForeslåVedtakBuilder(hentVedtakbrevtekstResponse: List<AvsnittDto>) {
+
+    private val perioderMedTekst: MutableList<PeriodeMedTekstDto> = mutableListOf()
+
+    init {
+        hentVedtakbrevtekstResponse.filter { it.avsnittstype == Avsnittstype.PERIODE }.forEach {
+            perioderMedTekst.add(
+                    PeriodeMedTekstDto(
+                            periode = PeriodeDto(fom = it.fom!!,
```

Comment:
Det kaster NullPointerException når it.fom og it.tom er null, jeg tror du må sjekke det først og så kan du opprette liste

Reply:
La  it.fom != null og it.tom != null i filteret

Reply:
mangler formattering. Når du har nested loop/iterasjon, er det bedre å bruke navn istedenfor it i nested iterasjon

Reply:
har endret formatering og fikset nested loop